packages/block-library/src/rss/index.js

```diff
@@ -20,5 +20,10 @@ export const settings = {
 		align: true,
 		html: false,
 	},
+	example: {
+		attributes: {
+			feedURL: 'https://wordpress.org',
```

Comment:
```suggestion
			feedURL: __( 'https://wordpress.org' ),
```

Reply:
> Suggested change 
> 			feedURL: 'https://wordpress.org',
> 			feedURL: __( 'https://wordpress.org' ),

URL's don't need to be translated. 👌

Reply:
It's usually helpful to translate users to redirect users to the homepage of their locale website like https://hi.wordpress.org/ if they're using WP in that locale.